.rubocop_todo.yml

```diff
@@ -0,0 +1,264 @@
+# This configuration was generated by
```

Comment:
getting rid of this whole file and defining what we want + cleaning up everything else would be nice too (maybe start by ignoring a bunch of rules to keep the diff low)

Reply:
starting point can be https://github.com/zendesk/kennel/blob/master/.rubocop.yml